csi/src/server.rs

```diff
@@ -237,12 +237,29 @@ async fn main() -> Result<(), String> {
         format!("{}:{}", endpoint, GRPC_PORT)
     };
 
-    let _ = tokio::join!(
-        CsiServer::run(csi_socket, node_name),
-        MayastorNodePluginGrpcServer::run(
+    let mut signal_term = tokio::signal::unix::signal(
+        tokio::signal::unix::SignalKind::terminate(),
+    )
+    .expect("Failed to egister handler for SIGTERM");
+    let mut signal_int = tokio::signal::unix::signal(
+        tokio::signal::unix::SignalKind::interrupt(),
+    )
+    .expect("Failed to register handler for SIGINT");
+
+    tokio::select! {
```

Comment:
In line with comments elsewhere, I do not understand or see, why it helps to override the signal handler with nothing.